configurations/__init__.py

```diff
@@ -32,3 +32,4 @@ def setup(app=None):
     in your docs' ``conf.py``.
     """
     _setup()
+    return {'version': __version__, 'parallel_read_safe': True}
```

Comment:
I see. Good point! I'll get to it. This well mean, that I will have to create a new major release tho, since it will be a breaking change.